src/schema/v2/me/task.ts

```diff
@@ -29,10 +28,6 @@ export const TaskType = new GraphQLObjectType<any, ResolverContext>({
     },
     title: { type: new GraphQLNonNull(GraphQLString) },
     message: { type: new GraphQLNonNull(GraphQLString) },
-    actionMessage: {
-      type: new GraphQLNonNull(GraphQLString),
-      resolve: ({ action_message }) => action_message,
-    },
```

Comment:
I don't think we can remove a field once it has been released 😅 since it will break the schema, we can only mark it as deprecated

Reply:
We had a similar discussion here https://github.com/artsy/metaphysics/pull/5690
It is my understanding that if no clients are using it then we can remove, is that not right?

Reply:
good point! and I think it's app only, so it makes sense to remove it since it hasn't been released yet 👍